examples/java/src/main/java/org/apache/beam/examples/complete/AutoComplete.java

```diff
@@ -386,7 +385,8 @@ public FormatForDatastore(String kind, String ancestorKey) {
     @ProcessElement
     public void processElement(ProcessContext c) {
       Entity.Builder entityBuilder = Entity.newBuilder();
-      Key key = makeKey(makeKey(kind, ancestorKey).build(), kind, c.element().getKey()).build();
+      com.google.datastore.v1.Key key =
+          makeKey(makeKey(kind, ancestorKey).build(), kind, c.element().getKey()).build();
 
```

Comment:
the changes in this file don't seem necessary. We've got rid of an import but introduced a fully qualified class name instead.

Reply:
The mentioned Key was conflicting with our DoFn.Key interface.